README.md

```diff
@@ -44,6 +44,18 @@ In most Linux distributions your user won't have access to serial interfaces by
 sudo usermod -aG dialout ${USER}
 ```
 
+On Debian, post-installation errors can be prevented by making sure the directory `/usr/share/desktop-directories` exists. To make sure it exists, run the following command before installing Betaflight Configurator:
```

Comment:
```suggestion
Post-installation errors can be prevented by making sure the directory `/usr/share/desktop-directories` exists. To make sure it exists, run the following command before installing Betaflight Configurator:
```

We prefer a more generic description, as this is true for other Linux flavors.

Reply:
I'll fix this :+1:

Reply:
Done :white_check_mark: